test/integration/TokenRejectIntegrationTest.js

```diff
@@ -0,0 +1,834 @@
+import { expect } from "chai";
+import {
+    AccountAllowanceApproveTransaction,
+    AccountBalanceQuery,
+    AccountCreateTransaction,
+    AccountUpdateTransaction,
+    Hbar,
+    NftId,
+    PrivateKey,
+    TokenCreateTransaction,
+    TokenMintTransaction,
+    TokenPauseTransaction,
+    TokenRejectTransaction,
+    TokenType,
+    TransactionId,
+    TransferTransaction,
+} from "../../src/exports.js";
+import IntegrationTestEnv from "./client/NodeIntegrationTestEnv.js";
+
+describe("TokenRejectIntegrationTest", function () {
+    let env, tokenId, receiverId, receiverPrivateKey;
+
+    beforeEach(async function () {
```

Comment:
This `beforeEach` is for fungible token tests, right? If true - then this block should be inside `describe("Fungible Tokens", function () {...}`